NitroxModel/DataStructures/GameLogic/AbsoluteEntityCell.cs

```diff
@@ -50,7 +29,28 @@ public AbsoluteEntityCell(Vector3 worldSpace, int level)
             BatchId = Int3.Floor(localPosition);
 
             Vector3 cell = (localPosition - BatchId.ToVector3()) * GetCellsPerBlock();
-            CellId = Int3.Floor(new Vector3(cell.x + 0.0001f, cell.y + 0.0001f, cell.z + 0.0001f));
+            CellId = Int3.Floor(new Vector3(cell.X + 0.0001f, cell.Y + 0.0001f, cell.Z + 0.0001f));
+        }
+
+        [ProtoMember(1)]
+        public Int3 BatchId { get; }
+
+        [ProtoMember(2)]
+        public Int3 CellId { get; }
+
+        [ProtoMember(3)]
+        public int Level { get; }
+
+        private Int3 BatchPosition => BatchId * Map.Main.BatchSize - Map.Main.BatchDimensionCenter;
+        public Int3 Position => BatchPosition + CellId * GetCellSize();
+
+        public Int3 Center
+        {
+            get
+            {
+                Int3 cellSize = GetCellSize();
+                return BatchPosition + CellId * cellSize + (cellSize >> 1);
+            }
         }
```

Comment:
Same here.  Can we make the auto formatter less aggressive?  it is nuking a lot of git blame for no purpose.  or modify it to keep properties and members above constructor.